### ISSUE TITLE
About the fold [0,1,2,3,4,5]

### ISSUE THREAD

**Issue by Jx-Tan** [282857341/nnFormer#49]
In the README.md, the command is used fold 0, Can I use other parameters, such as fold 1, fold 2, fold 3...? Does the modification affect the GPU memory and running time？

**Comment by 282857341** [comment]
The modification  won't  affect the GPU memory and running time

If you want to use other folds, it is best to delete the following code, otherwise the data division of all fold parameters will be the same.

https://github.com/282857341/nnFormer/blob/80fc0e04b86e4e339f171ef2981d3ee425715905/nnformer/training/network_training/nnFormerTrainerV2_ACDC.py#L311

**Comment by Jx-Tan** [comment]
Thanks a lot. Dose the different folds affect the result of segmentation?

**Comment by 282857341** [comment]
yes it will affect

**Comment by Jx-Tan** [comment]
So does its impact on the results depend on the data set and the task itself？

**Comment by 282857341** [comment]
Yes.
The impact on synapse is great, because its validation set is the test set. Maybe the impact on acdc may be relatively small.

**Comment by Jx-Tan** [comment]
Thanks
